src/tests/BuildWasmApps/testassets/BlazorApp/BlazorApp.csproj

```
@@ -1,13 +1,13 @@
 <Project Sdk="Microsoft.NET.Sdk.BlazorWebAssembly">
 
   <PropertyGroup>
-    <TargetFramework>net5.0</TargetFramework>
+    <TargetFramework>$(NetCoreAppCurrent)</TargetFramework>
```

A test is failing: https://helix.dot.net/api/2019-06-17/jobs/afc81c2d-b627-4f7b-9a1c-0d292ec73389/workitems/Workloads-.BlazorWasmTests/console.

Looks like needs to be hardcoded, since this project is published as by the test (BlazorWasmTests) on a remote machine where we don't have props file containing `NetCoreAppCurrent` definition:
```suggestion
    <TargetFramework>net7.0</TargetFramework>
```
(or net6.0 if it fails with 7 for some reason)

---

Thanks